test/JDBC/expected/restricted_objects-vu-verify.out

```diff
@@ -0,0 +1,61 @@
+-- tsql
+USE msdb
+GO
+
+ALTER LOGIN babel_5146_user_l1 WITH PASSWORD = 'abc';
+GO
+
+GRANT CONNECT TO guest;
+GO
+
+-- tsql user=babel_5146_user_l1 password=abc
+-- should throw error
+USE msdb
+GO
+
+EXEC dbo.babel_5146_prepare_p1;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: must be owner of function fn_syspolicy_is_automation_enabled)~~
+
+
+EXEC dbo.babel_5146_prepare_p2;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: must be owner of view syspolicy_configuration)~~
+
+
+EXEC dbo.babel_5146_prepare_p3;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: must be owner of view syspolicy_system_health_state)~~
+
+
+-- psql user=babel_5146_user_l1 password=abc
+-- should throw error
+DROP FUNCTION msdb_dbo.fn_syspolicy_is_automation_enabled;
```

Comment:
1. Please add all 11 object test cases explicitly.
2. Add a testcase when current user is dbo/or member of db_owner/db_ddladmin as well.
3. Add a testcase for case sensitivity.
4. Add a testcase for superuser(if it is not possible to add it in automated tests, please test it manually and add to description.)

Reply:
Tests for other objects which are not included in this PR have already been added to `restricted_procedures` test file. This change is only for `msdb` objects.